pkg/virt-operator/resource/generate/components/daemonsets.go

```diff
@@ -22,10 +22,12 @@ const (
 	kubeletPodsPath = "/var/lib/kubelet/pods"
 )
 
-func NewHandlerDaemonSet(namespace string, repository string, imagePrefix string, version string, launcherVersion string, productName string, productVersion string, productComponent string, pullPolicy corev1.PullPolicy, migrationNetwork *string, verbosity string, extraEnv map[string]string) (*appsv1.DaemonSet, error) {
+func NewHandlerDaemonSet(namespace string, imageName string, repository string, imagePrefix string, version string, launcherImage string, launcherVersion string, productName string, productVersion string, productComponent string, pullPolicy corev1.PullPolicy, migrationNetwork *string, verbosity string, extraEnv map[string]string) (*appsv1.DaemonSet, error) {
 
 	deploymentName := VirtHandlerName
-	imageName := fmt.Sprintf("%s%s", imagePrefix, deploymentName)
+	if imageName == "" {
```

Comment:
We prefer not to re-use a variable that was passed as an argument, can we do something like this instead:
```
imageName := VirtHandlerImageName
if len(imageName) == 0 {
  imageName = fmt.Sprintf("%s%s", imagePrefix, deploymentName) 
}
```